components/decimal/src/provider.rs

```diff
@@ -117,13 +94,13 @@ pub struct GroupingSizesV1 {
 )]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 pub struct DecimalSymbolsV1<'data> {
-    /// Prefix and suffix to apply when a negative sign is needed.
+    /// Pattern to apply when a negative sign is needed.
     #[cfg_attr(feature = "serde", serde(borrow))]
-    pub minus_sign_affixes: AffixesV1<'data>,
+    pub minus_sign_pattern: Option<SinglePlaceholderPattern<Cow<'data, str>>>,
```

Comment:
strictly speaking this should use a pattern type that requires exactly one placeholder

Reply:
When you make `.suffix()` return an `Option`, you can debug_assert that it is `Some`.